env0/resource_environment_import.go

```diff
@@ -126,9 +131,17 @@ func resourceEnvironmentImportUpdate(ctx context.Context, d *schema.ResourceData
 	return nil
 }
 
-// should not actually delete the environment import
-// this resource is used to populate the environment import wizard data
-// we don't want to delete the environment import after it's been created outside the wizard
 func resourceEnvironmentImportDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	// don't delete if soft delete is set
+	if softDelete, ok := d.GetOk("soft_delete"); ok && softDelete.(bool) {
```

Comment:
same here.. once you have a default.. just get the value...

Reply:
[eed48af](https://github.com/env0/terraform-provider-env0/pull/866/commits/eed48af5b5c5d63a6c376f7d7d746f992270db90)